sim/mage/arcane_blast.go

```diff
@@ -28,6 +27,7 @@ func (mage *Mage) registerArcaneBlastSpell() {
 	})
 
 	actionID := core.ActionID{SpellID: 42897}
+	totalDiscount := 1 - .01*float64(mage.Talents.ArcaneFocus+mage.Talents.Precision)
```

Comment:
I think I added precision already, in talents.go. You might need to change that so it isn't double-dipped.

Reply:
I want a better way to do this, but in essence my ModifyCast function needs to have the correct BaseManaCost after precision applies, but precision as a pseudostat.CostMultiplier only applies later. Without the discount the cost of AB is quite a bit too high because of the multiplication for the number of stacks. Also technically precision and arcane focus are supposed to be additive, but it really doesn't matter except for AB because the values aren't big enough to even notice a difference.

Anyway the way I solved the issue here is that I double count precision and then I subtract out one of the instances of precision in the cast.Cost call.

What would be best is if I had a way for precision to reduce all base costs of all the mage spells and additively stack with arcane focus, but that seemed like a lot of potential changes for some very minor mana diffferences.